### PR TITLE
query time range tests

### PR DESCRIPTION
**Pull request by rmakulov**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 1]
```diff
@@ -549,4 +551,104 @@ public void testMillisecondsUnsupported() throws Exception {
         JSONAssert.assertEquals("{\"error\":\"org.codehaus.jackson.map.JsonMappingException: Expected '-' character but found '5' (through reference chain: com.axibase.tsd.model.api.ApiTimeSeriesModel[\\\"data\\\"]->com.axibase.tsd.model.api.ApiTimeSeriesValue[\\\"d\\\"])\"}", response.readEntity(String.class), true);
     }
 
+    /* #3013 */
+    @Test
+    public void testQueryRangeIsLeftFromStorable() throws Exception {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: `isBeforeStorable`?

File: src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java [hunk 2]
```diff
@@ -549,4 +551,104 @@ public void testMillisecondsUnsupported() throws Exception {
...
+
+        SeriesQuery seriesQuery = new SeriesQuery(series.getEntity(), series.getMetric(), MIN_QUERYABLE_DATE, MIN_STORABLE_DATE);
+        List<Sample> data = executeQueryReturnSeries(seriesQuery).get(0).getData();
+
+        assertEquals("Not empty data for disjoint query and stored interval", 0, data.size());
+    }
+
+    /* #3013 */
+    @Test
+    public void testQueryRangeIsRightFromStorable() throws Exception {
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesInsertTest.java]: Query tests in SeriesInsertTest class?

**Comment by Sild** [comment]
ok

**Comment by KutuzovGregory** [comment]
ok

**Comment by unrealwork** [comment]
ok

**Pull request merged**